data/json/dreams.json

```diff
@@ -326,6 +377,15 @@
     "category": "LIZARD",
     "strength": 3
   },
+  {
+    "type": "dream",
+    "messages": [
+      "You have a strange dream about being the Easter Bunny.",
+      "You have a comforting and warm dream of living as a rabbit."
+    ],
+    "category": "LIZARD",
```

Comment:
Shouldn't this be "RABBIT"?

Reply:
Thanks! This was already merged so someone will have to create a new PR with the fixes.   Let me know if you want to that and if not I'll put them on my todo list.

Reply:
Oh, then I will do the PR, serves me as an excuse to update my master, just a question, NIGHT_RAIDER is supposed to have the same description as TUNNEL_FIGHTER? To know if I should change it too.

Reply:
no it should have different text .  I just forgot to update it.

Reply:
Done #51859.